projects/plugins/crm/src/automation/class-base-trigger.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Base Trigger implementation
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation;
+
+/**
+ * Base Trigger implementation
+ *
+ * @inheritDoc
+ */
+abstract class Base_Trigger implements Trigger {
+
+	/** @var mixed|string The trigger name */
```

Comment:
AFAIK then this will always be a string since we register triggers in code, right?

```suggestion
	/** @var string The trigger name */
	protected $name = '';
```